definitions/plistlib.d.ts

```diff
@@ -1,3 +1,5 @@
 declare module "plistlib" {
-	export function toString(data: any): string;
+    export function toString(data: any): string;
+    export function loadBuffer(data: NodeBuffer, callback: (err: any, plist: any) => void): void;
+    export function loadString(data: string, callback: (err: any, plist: any) => void): void;
```

Comment:
It is a 3rd party library. I'll have no idea until I read their implementation. I do not want to give promises I can not keep.